Makefile

```diff
@@ -27,6 +27,7 @@ PULL_SVCS = $(shell find ./services -type f -name 'docker-compose.yml' | sort -u
 
 # List of services to run
 START_SVCS = $(shell cat .services | grep -v \\\#)
+START_NEOFS = $(shell cat .services | grep -e .*chain -e ir -e storage -e basenet)
```

Comment:
What is we create a list of basic services, instead of doing exclusion?

Reply:
It's not exclusion. We `grep` morph_chain, chain, storage, ir, and basenet4 thus, we receive a list of basic services.

Reply:
The initial idea behind service directories is that they are independent and can be added by developer. If, as a developer, I add a new service, I also must edit the Makefile, and that's a bad thing to do. That's counterintuitive and leads to occasionally broken devenv.